frontend/packages/console-dynamic-plugin-sdk/src/extensions/pages.ts

```diff
@@ -14,7 +33,65 @@ export type StandaloneRoutePage = ExtensionDeclaration<
   }
 >;
 
+/** Adds new page to Console router. */
+export type RoutePage = ExtensionDeclaration<
+  'console.page/route',
+  {
+    /** The React component to load for the page. */
+    component: CodeRef<React.Component<RouteComponentProps>>;
+    /** The perspective ID to which this item belongs to. If not specified, contributes to all perspectives. */
+    perspective?: string;
+    /** Valid URL path or array of paths that `path-to-regexp@^1.7.0` understands. */
+    path: string | string[];
+    /** When true, will only match if the path matches the `location.pathname` exactly. */
+    exact?: boolean;
+    /** When true, a path that has a trailing slash will only match a location.pathname with a trailing slash. This has no effect when there are additional URL segments in the location.pathname. */
+    strict?: boolean;
```

Comment:
Please go over all usages of the current `Page/Route` extension and see if we really need those additional properties, like `strict` and `sensitive`.